Formula/llvm-arm-embedded.rb

```diff
@@ -0,0 +1,102 @@
+class LlvmArmEmbedded < Formula
+  desc "LLVM toolchain for Arm embedded targets"
+  homepage "https://github.com/ARM-software/LLVM-embedded-toolchain-for-Arm"
+  license all_of: [
+    { "Apache-2.0" => { with: "LLVM-exception" } },
+    "BSD-3-Clause",
+  ]
```

Comment:
Does this `BSD-3-Clause` come form `picolibc`? If so, it seems that it uses `BSD-2-Clause` as well: https://github.com/picolibc/picolibc#license

Reply:
Alright!  Should I include that too?

Reply:
Yes, please.